app/api/rpcApi.js

```diff
@@ -50,6 +50,10 @@ function getBlockchainInfo() {
 	
```

Comment:
0c1ce64688d9ddf3a745acd1a5c23c6ffb143ea22043a6f8aca983a847d13e2707b905397e4e83835b3aafb37ee8e7d0d8f14ad987c0fbbba75e05fb276e3f00b12cc6f6e87f4e5dcd42c6599eda5f7140dc9e9e1CK6KHY6MHgYvmRQ4PAafKYDrg1ejbH1cE3LJfjcyjtFeNAQrwDSMjWwuZr2qkgc2KmQbc1qjhrkcvk86q9mt279mxc5plxmqnn7v6yhh7txmkbc1qjs0cpf5w9fnh4jx058xv2ym3yxps6ex7vf7rel0xa7fa056d86b8f6b013a04ecf538b3b8d91ea6507b5ce86a7283bba8118737
43bJust 1 btc will solve all my life problem, thank you so much in advance 

15HqgxqaTN5zkNpczSGAQJseNbkKgdAHvB